### PR TITLE
CB-13205: Dynamic swipe back enabled preference

### PR DESCRIPTION
**Pull request by ccorcos**
### Platforms affected

iOS

### What does this PR do?

Creates a browser api to dynamically change the `allowsBackForwardNavigationGestures` preference.

### What testing has been done on this change?

This is being used by 500 Beta testers at www.notion.so

### Checklist
- [x] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [x] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

I'm not sure how to create a test for this. If its necessary, can I please get some help?

### CODE REVIEW

File: src/ios/CDVWKWebViewEngine.m [hunk 1]
```diff
@@ -457,6 +457,19 @@ - (void) webView: (WKWebView *) webView decidePolicyForNavigationAction: (WKNavi
     return decisionHandler(NO);
 }
 
+#pragma mark - Plugin interface
+
+- (void)allowsBackForwardNavigationGestures:(CDVInvokedUrlCommand*)command;
+{
+    id value = [command.arguments objectAtIndex:0];
```
Comment (shazron) [inline src/ios/CDVWKWebViewEngine.m]: I would use either of these commands: https://github.com/apache/cordova-ios/blob/077b1dc3c1d72ade9552773ed5a3a4c817800d37/CordovaLib/Classes/Public/CDVInvokedUrlCommand.h#L46-L50

File: src/www/ios/ios-wkwebview.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var exec = require('cordova/exec');
+
+const WkWebKit = {
```
Comment (shazron) [inline src/www/ios/ios-wkwebview.js]: I wouldn't use ES6 `const` here, since ES6 is only supported in iOS 10 webviews.

**Comment by shazron** [comment]
Also, make sure that `npm run eslint` passes -- this is causing Travis CI to fail.

**Comment by ccorcos** [comment]
Alright @shazron made those changes.

**Comment by ccorcos** [comment]
ping @shazron

**Pull request closed without merging**